packages/tracking/src/Tracking.php

```diff
@@ -5,10 +5,11 @@
 
 namespace Automattic\Jetpack;
 
-require_once dirname( dirname( dirname( __DIR__ ) ) ) . '/_inc/lib/tracks/client.php';
+use Automattic\Jetpack\Tracks\Client;
 
 class Tracking {
-	static $product_name = 'jetpack';
+	static $product_name  = 'jetpack';
+	const PACKAGE_VERSION = '1.0';
```

Comment:
Another source of truth for versioning...I know `filemtime` has caused trouble in the past, but it's a more reliable way to both leverage browser caching and bust the cache automatically. Could we have a wrapper around it that would fall back to `time()` if `filemtime` fails, and avoid relying on such statically defined versions completely?